frontend/lib/src/components/input/date_picker_field.dart

```diff
@@ -78,6 +78,7 @@ class DatePickerField extends StatelessWidget {
               validator: validator,
               initialEntryMode: DatePickerEntryMode.calendar,
               initialValue: initialValue,
+              lastDate: DateTime.now(),
```

Comment:
make it a prop to make it more dynamic. Other usage for this component doesn't necessarily need to disable all future dates.